examples/src/main/java/com/teamdev/jxbrowser/examples/PrintSettings.java

```diff
@@ -53,6 +53,12 @@ public static void main(String[] args) {
                     .orElseThrow(IllegalStateException::new);
             PrintJob<HtmlSettings> printJob = printer.printJob();
             printJob.settings()
+                    // Configure header with custom text and document title.
```

Comment:
Let's remove comments, we don't use them on other lines. And, IMHO, the code explains itself.